catlearn/categorical_model.py

```diff
@@ -95,7 +92,8 @@ def __init__(
         """
         assert epsilon > 0., "epsilon should be strictly positive"
 
-        self._relation_models = relation_models
+        self._relation_model = relation_model
+        self._label_universe = label_universe
```

Comment:
add a property to access a MappingProxyType to label_universe (see relations property in previous version)

Reply:
Done